src/status_im2/contexts/wallet/address_watch/view.cljs

```diff
@@ -13,13 +13,10 @@
 
 (defn view-internal
   []
-  (let [top         (safe-area/get-top)
-        bottom      (safe-area/get-bottom)
+  (let [bottom      (safe-area/get-bottom)
```

Comment:
do we need bottom here, should this also be used on insets on screens.cljs file instead? 🤔